src/stampy_nlp/utilities/coda_utils.py

```diff
@@ -25,7 +25,7 @@ def fetch_all_rows():
     headers: object = {'Authorization': f'Bearer {get_coda_token()}'}
     params: object = {
         'useColumnNames': True,
-        'limit': 1000
+        'limit': 10000
```

Comment:
We're nearing the 1000 limit. Increasing it by 10x. Maybe there's a better way to do this?

Reply:
yup. Pagination, which the following lines do :) Coda ignores anything over 500, anyway

Reply:
Should I just remove that line entirely then?

Reply:
might as well